docs/data/charts/bars/bars.md

```diff
@@ -61,3 +61,7 @@ Bar charts can be rendered with a horizontal layout by providing the `layout="ho
 If you're using [composition](/x/react-charts/#multiple-charts), you should set the property `layout: 'horizontal'` to each bar series object.
 
 {{"demo": "HorizontalBars.js"}}
+
+## Animation
```

Comment:
```suggestion
## Animation

To animate the creation and update of your chart use the `animate` prop.
When set to `true` it activate animation using `@react-spring/web`.

```jsx
// For a single component chart
<BarChart {/* ... */} animate />

// For a composed chart
<ResponsiveChartContainer>
  {/* ... */}
  <BarPlot  animate />
</ResponsiveChartContainer>
```

When you activate animation, use their [`useReduceMotion`](https://www.react-spring.dev/docs/utilities/use-reduced-motion) hook for accessibility reasons.